app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -92,11 +88,6 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         setTaskDescription(new ActivityManager.TaskDescription(appName, appIcon, recentsBackgroundColor));
 
-        long accountId = getIntent().getLongExtra("account", -1);
-        if (accountId != -1) {
-            accountManager.setActiveAccount(accountId);
-        }
-
```

Comment:
`"account"` is not used anywhere else in the app. The real account switching code is in `MainActivity`